lib/ctcs2_to_junit.pm

```diff
@@ -99,15 +99,15 @@ sub generateXML {
             name      => $test,
             status    => $case_status,
             time      => $test_results{$test}->{time});
-        if ($case_status eq 'failure' || $case_status eq 'skipped') {
+        if (get_var("QA_TESTSUITE") eq "" && ($case_status eq 'failure' || $case_status eq 'skipped')) {
```

Comment:
looks strange, what are you trying to check here? that variable QA_TESTSUITE is not defined? better to do it with `!get_var('QA_TESTSUITE')`. also, there is a `check_var` function which is another option to use here.

Reply:
Gotta, forgot to use this. I will change to `!get_var('QA_TESTSUITE')`. Thanks a lot!